packages/ember-engines/package.json

```diff
@@ -103,22 +102,17 @@
     "calculate-cache-key-for-tree": "^2.0.0",
     "ember-asset-loader": "^1.0.0",
     "ember-cli-babel": "^7.18.0",
+    "ember-cli-htmlbars": "^5.3.2",
```

Comment:
maybe go v6?

Reply:
waiting on ci to see it green then can bump this to 6 and node to 18 in one go